db/migrate/20160203102911_migrate_users_to_have_role.rb

```diff
@@ -0,0 +1,23 @@
+class MigrateUsersToHaveRole < ActiveRecord::Migration
+  def up
+    Comfy::Cms::User.all.each do |user|
+      user.update_attribute(:role, new_role_for(user))
+    end
+  end
+
+  def down
+    Comfy::Cms::User.all.each do |user|
+      user.update_attribute(:admin, user.admin?)
+    end
+  end
+
+  private
+
+  def new_role_for(user)
+    if user.read_attribute(:admin)
+      Comfy::Cms::User.roles[:admin]
+    else
+      Comfy::Cms::User.roles[:user]
+    end
+  end
+end
```

Comment:
We could move it out to a separate task, but the reason for it being part of this is to prevent a misunderstanding in behaviour.

The effect of using `enum role` with an `'admin'` value, is that the `Comfy::Cms::User` obtains an `#admin?` method, which led to a little bit of confusion when an `admin` attribute is left lying around on the model, as it also dynamically creates an `#admin?` method. 

In our scenario, the `#admin?` method for our new role functionality appears to be the last method loaded and therefore it is its functionality we would be using. However, without digging further, I couldn't be confident in saying that this method would _always_ be the last one loaded. And it also means we have duplicate data lingering within the application.

In this case, my preference would be to keep things as is, but we can change it if you think we should?